react/features/rtcstats/middleware.ts

```diff
@@ -47,25 +46,10 @@ MiddlewareRegistry.register((store: IStore) => (next: Function) => (action: AnyA
         }
         break;
     }
-    case TRACK_ADDED: {
-        if (isRTCStatsEnabled(state)) {
-            const jitsiTrack = action?.track?.jitsiTrack;
-            const { ssrc, videoType } = jitsiTrack || { };
-
-            // Remote tracks store their ssrc in the jitsiTrack object. Local tracks don't. See getSsrcByTrack.
-            if (videoType && ssrc && !jitsiTrack.isLocal() && !jitsiTrack.isAudioTrack()) {
-                RTCStats.sendVideoTypeData({
-                    ssrc,
-                    videoType
-                });
```

Comment:
Is this also handled in ljm?

Reply:
I don't see any use for this data. The videoType of the remote track never changes so I do not see any value in tracking/sending this.